### DISCUSSION TITLE
Windows Server, version 1909 no longer supported by .NET

### DISCUSSION THREAD

**Discussion by mthalman** [dotnet/dotnet-docker#3687] [Announcements]
# Windows Server, version 1909 no longer supported by .NET

Windows Server, version 1909 is no longer supported. As a result, this version of Windows is no longer supported by .NET and we will no longer produce .NET container images for this Windows version.

Support statements:

* [Currently supported .NET Core Docker images](https://hub.docker.com/_/microsoft-dotnet-core)
* [.NET Core Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core)
* [Windows Server, version 1909 Release Information](https://docs.microsoft.com/windows/release-information/status-windows-10-1909)

## Details

Docker tags that contain `nanoserver-1909` are no longer being produced. We will not delete existing images or those tags. However, they contain unsupported builds of Nano Server and .NET.

## Pulling manifest tags

These changes also affect manifest tags (AKA “multi-arch tags“), such as `5.0`, `5.0.0`. These are tags that are supported on multiple operating systems, and do not contain the operating system in them. As a result of no longer producing new images for Windows Server, version 1909, manifest tags no longer include entries for Windows 10, version 1909, and Windows Server, version 1909. That means that .NET manifest tags are no longer usable on those Windows versions.

If you pull a manifest tag on one of the Windows versions described, you may get one of the following error messages:

* `a Windows version 10.0.XXXXX-based image is incompatible with a 10.0.18363 host`
* `no matching manifest for windows/amd64 in the manifest list entries`
* `no matching manifest for unknown in the manifest list entries`

If you need to still use Nano Server, version 1909 images (even though it is no longer supported), you will need to use tags that include the Nano Server version (these are non-manifest tags), for example `5.0-nanoserver-1909`.

**Comment by ghost** [comment]
I couldn't figure out the best area label to add to this issue. If you have write-permissions please help me learn by adding exactly one area label.
